File: api-sintec/src/main/java/br/com/sintec/model/RetornoResposta.java
package br.com.sintec.model;

import java.time.LocalDateTime;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.persistence.Table;

import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@ToString
@Getter
@Setter
@Entity
@Table(name="SERV_RETORNO_RESPOSTA")
public class RetornoResposta {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@EqualsAndHashCode.Include
	@Column(name = "ID")
	private Long id;
	@ManyToOne
	@JoinColumn(name="ID_SERVICO")
	private Servico servico;
	@ManyToOne
	@JoinColumn(name="ID_PERGUNTA")
	private Pergunta pergunta;
	@ManyToOne
	@JoinColumn(name="ID_ALTERNATIVA")
	private Alternativa alternativa;
	@Column(name = "CD_LATITUDE")
	private String latitude;
	@Column(name = "CD_LONGITUDE")
	private String longitude;
	@Column(name = "DS_RESPOSTA")
	private String resposta;
	@Column(name = "DT_EXECUCAO")
	private LocalDateTime dataExecucao;
}

File: MOBILE/app_sintec/lib/shared/model/retorno_resposta.dart
class RetornoResposta{
  int? id;
  int? idServico;
  int? idPergunta;
  int? idAlternativa;
  String? longitude;
  String? latitude;
  String? resposta;
  String? dataExecucao;
  int? sincronizado;

  RetornoResposta({
    this.id,
    this.idServico,
    this.idPergunta,
    this.idAlternativa,
    this.longitude,
    this.latitude,
    this.resposta,
    this.dataExecucao,
    this.sincronizado
});

  factory RetornoResposta.fromSQLite(Map map) {
    return RetornoResposta(
      id: map["id"],
      idServico: map["id_servico"],
      idPergunta: map["id_pergunta"],
      idAlternativa: map["id_alternativa"],
      longitude: map["longitude"],
      latitude: map["latitude"],
      resposta: map["resposta"],
      dataExecucao: map['data_execucao'],
      sincronizado: map["sincronizado"]
    );
  }

  static List<RetornoResposta> fromSQLiteList(List<Map> listMap) {
    List<RetornoResposta> listaRegional = [];
    for (Map item in listMap) {
      listaRegional.add(RetornoResposta.fromSQLite(item));
    }
    return listaRegional;
  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['idServico'] = this.idServico;
    data['idPergunta'] = idPergunta ;
    data['idAlternativa'] = this.idAlternativa;
    data['longitude'] = this.longitude;
    data['latitude'] = this.latitude;
    data['resposta'] = this.resposta;
    data['dataExecucao'] = this.dataExecucao;
    return data;
  }
}